build.assets/webapps/update-teleport-webassets.sh

```diff
@@ -74,7 +74,7 @@ git submodule update --init --recursive
 # set variables based on context from webapps checkout
 BRANCH=$(git rev-parse --abbrev-ref HEAD)
 COMMIT=$(git rev-parse --short HEAD)
-COMMIT_DESC=$(git log --decorate=off --oneline -1)
+COMMIT_DESC=$(git log --decorate=off --oneline -1 | sed -E 's.(#[0-9]+).gravitational/webapps\1.g')
```

Comment:
Can you add a comment here as to the why this sed section exists? I could just see it being a bit cryptic to some future editor.